tests/functional/network_policy/test_network_policy.py

```
@@ -0,0 +1,453 @@
+            logger.warning("No tools pod found, trying operator pod")
+            operator_pods = get_pods_having_label(
+                "app=rook-ceph-operator", namespace=namespace
+            )
+            assert operator_pods, "No tools or operator pod found for DNS test"
+            verify_dns_from_pod(
+                operator_pods[0]["metadata"]["name"], namespace=namespace
+            )
+
+    @tier1
+    @pytest.mark.polarion_id("OCS-6905")
+    def test_inter_component_communication(self, network_policies_present):
```

Ruff (0.16.0)

[warning] Unused method argument: `network_policies_present`

(ARG002)